Gemfile

```diff
@@ -16,6 +16,10 @@ gem 'turbolinks'
 gem 'uglifier', '>= 1.3.0'
 gem 'unf'
 gem 'will_paginate', '~> 3.0'
+gem 'wikipedia-client'
+gem 'rest-client'
+gem 'addressable'
```

Comment:
I'm ~~not~~ gonna be really picky, but can we keep the Gemfile in alphabetical order? :grin: